packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -35,14 +39,15 @@ class _AccountPictures extends StatelessWidget {
             children: (otherAccountsPictures ?? <Widget>[]).take(3).map<Widget>((Widget picture) {
               return Padding(
                 padding: const EdgeInsetsDirectional.only(start: 8.0),
-                child: Semantics(
-                  container: true,
-                  child: Container(
+                child: Padding(
                   padding: const EdgeInsets.only(left: 8.0, bottom: 8.0),
-                    width: 48.0,
-                    height: 48.0,
-                    child: picture,
-                 ),
+                  child: Semantics(
```

Comment:
Shouldn't the semantics wrap one of the paddings as well to reach the minimum tap target size of 48x48 on mobile?

Reply:
Yep, fixed!